docker/girder_worker.Dockerfile

```diff
@@ -59,5 +62,5 @@ USER worker
 ENV VIAME_TRAINED_PIPELINES_PATH /home/worker/trained
 RUN mkdir -p ${VIAME_TRAINED_PIPELINES_PATH}
 
-ENTRYPOINT ["/tini", "--", "python3.7", "-m", "girder_worker" ]
-CMD [ "-l", "info" ]
+ENTRYPOINT ["/tini", "--"]
+CMD ["/home/provision/girder_worker_entrypoint.sh"]
```

Comment:
You have to be careful when running shell scripts in docker, because you can mess up signal handling.

Tini traps signals and sends them through to the child.  Before, the child process was python. Now the child process is a bash shell, which I do not think will forward signals from tini to girder worker.

It's kinda fuzzy what happens here when you do this, but https://github.com/krallin/tini/issues/8 is a good starting place.  also https://unix.stackexchange.com/questions/146756/forward-sigterm-to-child-in-bash 

See below.

Reply:
Good catch, not something I was aware of.